paradox-menu.el

```diff
@@ -553,7 +553,19 @@ Defines a function called paradox-sort-by-NAME."
 (paradox--define-sort "Package")
 (paradox--define-sort "Status")
 (paradox--define-sort paradox--column-name-star "*")
+(paradox--define-sort "Version")
 (declare-function paradox-sort-by-package "paradox-menu")
+(declare-function paradox-sort-by-version "paradox-menu")
+
+(defun paradox--version-predicate (A B)
```

Comment:
avoid upper-case names for function arguments. `package-a` and `package-b` would be more appropriate here.